try {
  console.log("Iniciando processamento.");
} catch (error) {
  console.error(error.message);
} finally {
  console.log("Fim do processo!");
}
